src/App.js

```diff
@@ -14,11 +15,86 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  }
+
+  selectGood = (good) => {
+    this.setState({ selectedGood: good });
+  }
+
+  unselectGood = () => {
+    this.setState({ selectedGood: null });
+  }
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="App column is-one-third is-centered">
+        <div className="panel">
+          <div
+            className="
+              panel-heading
+              is-flex
+              is-justify-content-space-between
+            "
+          >
+            <h1 className="title is-5">
+              {
+                selectedGood
+                  ? `${selectedGood} is selected` : 'No goods selected'
+              }
+            </h1>
+            <button
+              type="button"
+              className={
+                className(
+                  'button',
+                  { 'is-hidden': !selectedGood },
+                )
+              }
+              onClick={() => this.unselectGood()}
```

Comment:
unselectGood не приймає параметрів тому стрілочнку функцію можна не писати
```suggestion
              onClick={this.unselectGood}
```